### PR TITLE
Update edge-environmental-requirements.md

### PR DESCRIPTION
**Pull request by toninofilip**
As edited: rules in rows 366 and 367 showing both TCP and UDP port range 50000-59999 as destination, are incorrect. The rules are optional (if firewall requires specification of source ports) and in that case destination ports are TCP 443 and UDP 3478 (instead of 50000-59999 as stated). Source ports are TCP 50000-59999  and UDP 3478.  Otherwise, rules in rows 368 - 371 are sufficient to establish media path over Edge.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Skype/SfbServer/plan-your-deployment/edge-server-deployments/edge-environmental-requirements.md [hunk 1]
```diff
@@ -363,8 +363,8 @@ The Source IP address and Destination IP address will contain information for us
...
 |Web conferencing/PSOM(TLS)  <br/> |TCP  <br/> |443  <br/> |Any  <br/> |**Private IP using NAT:** Edge Server Web Conferencing Edge service <br/> **Public IP:** Edge Server Web Conferencing Edge service public IP address <br/> |Web conferencing media.  <br/> |
-|A/V/RTP  <br/> |TCP  <br/> |50000-59999  <br/> |**Private IP using NAT:** Edge Server A/V Edge service <br/> **Public IP:** Edge Server A/V Edge service public IP address <br/> |Any  <br/> |This is used for relaying media traffic.  <br/> |
-|A/V/RTP  <br/> |UDP  <br/> |50000-59999  <br/> |**Private IP using NAT:** Edge Server A/V Edge service <br/> **Public IP:** Edge Server A/V Edge service public IP address <br/> |Any  <br/> |This is used for relaying media traffic.  <br/> |
+|A/V/RTP  <br/> |TCP  <br/> |50000-59999*  <br/> |**Private IP using NAT:** Edge Server A/V Edge service <br/> **Public IP:** Edge Server A/V Edge service public IP address <br/> |Any  <br/> |This is used for relaying media traffic. *This rule is optional, and should be added only if firewall requires specification of source ports. In that case TCP 50000-59999 port range is specified as source port range, and destination port is TCP 443.  <br/> |
```
Comment (JohanFreelancer9) [inline Skype/SfbServer/plan-your-deployment/edge-server-deployments/edge-environmental-requirements.md:366]:
```suggestion
|A/V/RTP  <br/> |TCP  <br/> |50000-59999*  <br/> |**Private IP using NAT:** Edge Server A/V Edge service <br/> **Public IP:** Edge Server A/V Edge service public IP address <br/> |Any  <br/> |This is used for relaying media traffic. *This rule is optional, and should be added only if the firewall requires specification of source ports. In that case, port range TCP 50000-59999 is specified as the source port range, and the destination port is TCP 443.  <br/> |
```

File: Skype/SfbServer/plan-your-deployment/edge-server-deployments/edge-environmental-requirements.md [hunk 2]
```diff
@@ -363,8 +363,8 @@ The Source IP address and Destination IP address will contain information for us
...
+|A/V/RTP  <br/> |UDP  <br/> |3478*  <br/> |**Private IP using NAT:** Edge Server A/V Edge service <br/> **Public IP:** Edge Server A/V Edge service public IP address <br/> |Any  <br/> |This is used for relaying media traffic. *This rule is optional, and should be added only if firewall requires specification of source ports. In that case UDP 3478 port is specified as both source port and destination port.  <br/> |
```
Comment (JohanFreelancer9) [inline Skype/SfbServer/plan-your-deployment/edge-server-deployments/edge-environmental-requirements.md:367]:
```suggestion
|A/V/RTP  <br/> |UDP  <br/> |3478*  <br/> |**Private IP using NAT:** Edge Server A/V Edge service <br/> **Public IP:** Edge Server A/V Edge service public IP address <br/> |Any  <br/> |This is used for relaying media traffic. *This rule is optional, and should be added only if the firewall requires specification of source ports. In that case, port UDP 3478 is specified as both the source port and destination port.  <br/> |
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request closed without merging**